### ISSUE TITLE
Issue with integrated dynamics

### ISSUE THREAD

**Issue by VCROMBECQUE** [FTBTeam/FTB-Modpack-Issues#4675]
### Modpack

FTB Skies Expert

### Modpack version

1.0.3

### Describe the bug

When I open the integrated dynamics storage terminal, my client disconnects me from the server.

### Steps to reproduce

1. Place a vanilla
2. Connect this chest to the storage terminal with an item interface and a logic cable
3. Open the storage terminal

### Expected behaviour

Open storage terminal without crash

**Comment by Saereth** [comment]
I cant reproduce this and we're using them quite exensively. Can you send me your logs/debug logs for client and server if possible.

**Comment by Deathstar175** [comment]
I am getting the same issue whenever I open the ID terminal it crashes the client to main menu with this error message. 
![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/17036957/3fe1e132-a510-43a3-8350-ebc3e9722a8e)
It was working fine earlier, the server rebooted and 
then it started crashing.
Here is the client debug log. Server debug log is currently beyond my grasp.
[debug.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/14338791/debug.log)

**Comment by Saereth** [comment]
I had the testers try to reproduce this as well and no luck yet testing on the beta server, can you guys let me know what all you have hooked up on your id networks? Might be a certain storage block causing it. We really need a server log if possibe from either of you, would help.

**Comment by danyewest97** [comment]
I just made a GitHub account to comment on this thread, I have been having the same problem and it has happened with 2 different modpacks, Chosen's Modded Adventure and ATM9. I have had the exact same problem, and the same error message too. I think it has something to do with the storage terminal crafting and JEI, specifically auto-filled crafting with JEI, though it could be something else. The only solution I found was to restart the server, and the storage terminal started working again.
